src/createReconciler.lua

```diff
@@ -45,7 +45,13 @@ local function createReconciler(renderer)
 		local context = virtualNode.originalContext or virtualNode.context
 		local parentLegacyContext = virtualNode.parentLegacyContext
 
-		unmountVirtualNode(virtualNode)
+		if config.tempFixUpdateChildrenReEntrancy then
+			if not virtualNode.wasUnmounted then
+				unmountVirtualNode(virtualNode)
+			end
```

Comment:
So I take this to mean that the re-entrancy was, in some cases, causing the same node to be unmounted more than once, is that the case? Can we get a comment here that explains the scenario that might trigger it, the way we have with the other blocks you added?

Reply:
Added a comment for this

Reply:
I wonder if this could be related to #235? Would it fix that issue?

Reply:
I don't think this would fix that issue, seems to be more about extra renders than extra unmounts.